Adding more info about carbon emissions (#485)
huggingface/huggingface_hub: docs/hub/model-repos.md

Open Data Commons Public Domain Dedication and License | `pddl`
Lesser General Public License For Linguistic Resources | `lgpl-lr`

## How is the carbon footprint of my model calculated? 🌎

By taking into account the computing hardware, location, usage and training time, it's possible to provide an estimate of how much CO<sub>2</sub> was produced by the model.

The math is actually pretty simple! ➕

> First, you take the *carbon intensity* of the electric grid that is being used for the training -- this is how much CO<sub>2</sub> is produced by KwH of electricity used. This depends on the location where the hardware is located and the [energy mix](https://electricitymap.org/) used at that location -- whethere it's renewable energy like solar 🌞, wind 🌬️ and hydro 💧, or non-renewable energy like coal ⚫ and natural gas 💨. The more renewable energy is used, the less carbon intensive it is!

Reviewer: ```suggestion
First, you take the *carbon intensity* of the electric grid that is being used for the training -- this is how much CO<sub>2</sub> is produced by KwH of electricity used. This depends on the location where the hardware is located and the [energy mix](https://electricitymap.org/) used at that location -- whether it's renewable energy like solar 🌞, wind 🌬️ and hydro 💧, or non-renewable energy like coal ⚫ and natural gas 💨. The more renewable energy is used, the less carbon intensive it is!
```

> Then, you take the power consumption of the GPU  during training -- this is done using the `pynvml` library.

Reviewer: ```suggestion
Then, you take the power consumption of the GPU during training -- this is done using the `pynvml` library.
```

Finally, you multiply the power consumption and carbon intensity by the training time of the model, and you have an estimate of how much CO<sub>2</sub> was emitted.

> Keep in mind that this isn't an exact number, because there are other factors that come into play -- like the energy that's used for datacenter heating and cooling -- which will increase that number. But this will already give you a good idea of the scale of CO<sub>2</sub> emissions that your model is producing!

Reviewer: ```suggestion
Keep in mind that this isn't an exact number, because there are other factors that come into play -- like the energy that's used for data center heating and cooling -- which will increase carbon emissions. But this will already give you a good idea of the scale of CO<sub>2</sub> emissions that your model is producing!
```

To add **Carbon Emissions** metadata to your models:

> 1. If you are ussing **AutoNLP**, this is tracked for you 🔥

Reviewer: Maybe a green emoji 🌱  would be better? Fires make me think of burning things and releasing more CO2 😅

```suggestion
1. If you are using **AutoNLP**, this is tracked for you 🔥
```
